src/components/GirlsHooHack/FAQ/FAQ.tsx

```diff
@@ -1,9 +1,53 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { request } from 'graphql-request';
+import {Row, Col} from 'react-bootstrap';
+import { isMobile } from 'react-device-detect';
+import FAQComponent from './FAQComponent';
+/*
+import { memoryUsage } from 'node:process';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faPlus } from '@fortawesome/free-solid-svg-icons'
+import { faMinus } from '@fortawesome/free-solid-svg-icons'
+import {Collapse } from 'react-collapse';
+*/
 
 function FAQ() {
+
+  const [faq, setFaq] = useState<any[]>([]);
```

Comment:
avoid any types -- specify what the list is -- there should be a reference in the links pr